src/main/java/net/geforcemods/securitycraft/screen/BriefcasePasswordScreen.java

```diff
@@ -148,8 +148,13 @@ protected void actionPerformed(ClickButton button) {
 					CompoundNBT nbt = Minecraft.getInstance().player.inventory.getCurrentItem().getTag();
 					String code = keys[0] + "" + keys[1] + "" +  keys[2] + "" + keys[3];
 
-					if(nbt.getString("passcode").equals(code))
+					if(nbt.getString("passcode").equals(code)) {
+						if (!nbt.contains("owner")) {
+							nbt.putString("owner", Minecraft.getInstance().player.getName().getString());
+							nbt.putString("ownerUUID", Minecraft.getInstance().player.getUniqueID().toString());
+						}
```

Comment:
There's a problem with this. Any existing briefcase can be claimed by anyone knowing the code. I don't know if we can do anything about this, and if it is a big problem, but in any case it's something to keep in mind.

Also, please add a new line after the closing bracket.

(And holy shit what the fuck is the rest of that code in this method? I need to look at improving that...)

Reply:
Yeah, but I don't see much of a problem with that because the one that knows the code is usually the one that created the briefcase.
Also, I added the line you requested, you can close this review comment if you want to :-)